crates/symbolicator/src/services/shared_cache.rs

```diff
@@ -473,17 +492,22 @@ impl SharedCacheService {
             None => return false,
         };
         match res {
-            Ok(Some(_)) => {
+            Ok(Some(bytes)) => {
                 metric!(
-                    counter("services.shared_cache.store") += 1,
+                    counter("services.shared_cache.fetch") += 1,
                     "cache" => key.name.as_ref(),
                     "hit" => "true",
                 );
+                let bytes: i64 = bytes.try_into().unwrap_or(i64::MAX);
+                metric!(
+                    counter("services.shared_cache.fetch.bytes") += bytes,
```

Comment:
don’t we use a different metric type for these histograms?

Reply:
(turns out i had a half-written response here... i'm good at this)

The aim here is not to get an idea of the object sizes uploaded with percentiles of their sizes etc, object sizes already have a metric.  I'm only after the rate, bytes/second, of uploading and downloading to the shared cache so we know how much bandwidth we're spending on this.  This is to verify that we're not using too much of the available machine's bandwidth and that the throttling we have in place is sufficient to not disrupt symbolicator's normal work.